ingifet.py

```diff
@@ -74,10 +74,11 @@ def POST(self, id=None):
 
         if not form.validates():
             return render.edit_user(form, user,"")
-
-        for u in User.all():
-            if form.d.email == u.email:
-                return render.edit_user(form, user, 'NotUniqueEmail')
+        #check only if email changed
+        if user.email != form.d.email:
+            for u in User.all():
+                if form.d.email == u.email:
+                    return render.edit_user(form, user, 'NotUniqueEmail')
```

Comment:
This is not directly related to the patch, but wouldn't it be more efficient to query a User based on the form email instead of looping on all the user list ?

Reply:
I tought the same thing after making the pull request. I'll do the modification.

Reply:
Resolved by 8e87912